e2e/test/scenarios/permissions/sandboxes.cy.spec.js

```diff
@@ -221,6 +215,13 @@ describeEE("formatting > sandboxes", () => {
         },
       });
 
+      cy.sandboxTable({
+        table_id: PEOPLE_ID,
+        attribute_remappings: {
+          attr_uid: ["dimension", ["field", PEOPLE.ID, null]],
+        },
+      });
+
```

Comment:
This wasn't so much about the order, as much as it was about granting the "unrestricted" `view-data` permissions on a database level after sandboxing a table (basically making the previous action void).

This is how the test looked before the "split permissions"
https://github.com/metabase/metabase/blob/42ce333d5140ddd7381bba7fef8caefeb28a3f64/e2e/test/scenarios/permissions/sandboxes.cy.spec.js#L199-L214


1. We were staring with both "All users" and "Collection" groups data permissions set to "No self-service"
2. We then sandboxed the People table by matching the user attribute to the People.id
3. We then lifted the data restrictions for the three remaining tables (orders, products, reviews) but we didn't break sandboxing - this was targeted and selective

The test rewrite broke the sandboxing.
The correct payload should've been:
```js
cy.updatePermissionsGraph({
  [COLLECTION_GROUP]: {
    [SAMPLE_DB_ID]: {
      "view-data": {
        "PUBLIC": {
          [ORDERS_ID]: "unrestricted",
          [PRODUCTS_ID]: "unrestricted",
          [REVIEWS_ID]: "unrestricted",
          [PEOPLE_ID]: "sandboxed",
        }
      },
      "create-queries": {
        PUBLIC: {
          [ORDERS_ID]: "query-builder",
          [PRODUCTS_ID]: "query-builder",
          [REVIEWS_ID]: "query-builder",
        },
      },
    },
  },
});
```